source/renderer/app/components/staking/delegation-center/WalletRow.js

```diff
@@ -119,6 +120,48 @@ export default class WalletRow extends Component<Props, WalletRowState> {
     ...initialWalletRowState,
   };
 
+  stakePoolFirstTileRef: { current: null | HTMLDivElement };
+  stakePoolAdaSymbolRef: { current: null | HTMLDivElement };
+
+  constructor(props: Props) {
+    super(props);
+
+    this.stakePoolFirstTileRef = React.createRef();
+    this.stakePoolAdaSymbolRef = React.createRef();
+  }
+
+  componentDidUpdate() {
+    const {
```

Comment:
@yakovkaravelov can you please extract this logic into a separate method which then gets invoked in the `componentDidUpdate`?
That would be cleaner...